### PR TITLE
create moyo header

### PR DESCRIPTION
**Pull request by LBondarchuk**
- [DEMO LINK](https://LBondarchuk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://LBondarchuk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,75 @@
...
+
+.nav-item:last-child {
+  margin: 0;
+}
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
```
Comment (slavanosatskyi) [inline src/style.css:49]: usually it is better to apply font styles on a higher level, like in body tag

File: src/style.css [hunk 2]
```diff
@@ -1 +1,75 @@
...
+  max-width: 1200px;
+  min-width: 1024px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav {
+  margin-right: 50px;
```
Comment (slavanosatskyi) [inline src/style.css:28]: instead you can add paddings to your header

File: src/style.css [hunk 3]
```diff
@@ -1 +1,75 @@
...
+}
+
+body {
+  box-sizing: border-box;
+  height: 100vh;
+  background: #e5e5e5;
+}
+
+.logo-link {
+  margin-left: 50px;
```
Comment (slavanosatskyi) [inline src/style.css:13]: instead you can add paddings to your header

**Review by slavanosatskyi — CHANGES_REQUESTED** [review]

**Comment by slavanosatskyi** [comment]
Screenshot 2023-02-06 at 18 39 17
Please, pay attention that for really big screens header is not centered

File: src/style.css (hunk 2 above)
Comment (LBondarchuk) [reply src/style.css:28]: I tried to do this from the beginning. But npm test treated it as an error.

**Review by LBondarchuk — COMMENTED** [review]
I fixed the code.
But the padding did not change. The test then does not work correctly

File: src/style.css [hunk 4]
```diff
@@ -1 +1,76 @@
...
+  text-decoration: none;
+}
+
+.header {
+  margin: 0 auto;
+  max-width: 1200px;
+  min-width: 1024px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (polosanya) [inline src/style.css:25]: Pay attention to CSS properties order
https://9elements.com/css-rule-order/

**Review by polosanya — CHANGES_REQUESTED** [review]
Hi! I see that the problem from the previous comment was not fixed. Try to make the width 100% for the header.
image

**Pull request closed without merging**